include/inexor/vulkan-renderer/wrapper/gpu_texture.hpp

```diff
@@ -74,19 +74,19 @@ class GpuTexture {
     GpuTexture &operator=(const GpuTexture &) = delete;
     GpuTexture &operator=(GpuTexture &&) = delete;
 
-    [[nodiscard]] std::string name() const {
+    [[nodiscard]] std::string name() const noexcept {
```

Comment:
Shouldn't be `noexcept` since it can throw.

Reply:
I will drop the commit which introduces `noexcept` in get methods as it gives us no benefits.